mesos_slave/datadog_checks/mesos_slave/mesos_slave.py

```diff
@@ -127,7 +127,7 @@ def _get_json(self, url, timeout, verify, tags=None):
             if self.service_check_needed:
                 self.service_check(self.SERVICE_CHECK_NAME, status, tags=tags, message=msg)
                 self.service_check_needed = False
-            if status is AgentCheck.CRITICAL:
+            if status == AgentCheck.CRITICAL:
```

Comment:
Is this necessary? We should be careful about making changes during review period

Reply:
This only ever worked because of an implementation detail; CPython caches small integers.
```python
>>> 400 == 20 ** 2
True
>>> 400 is 20 ** 2
False
```

An object will always be equal to itself so `==` will always be the same as `is`, just not the other way around. I can remove this if you want though.